relay-protocol/src/lib.rs

```diff
@@ -23,17 +23,15 @@ mod macros;
 mod meta;
 mod size;
 mod traits;
-mod utils;
 mod value;
 
 pub use self::annotated::*;
-pub use self::condition::*;
+pub use self::condition::RuleCondition;
```

Comment:
In addition to this, we will need to update to `pub mod condition`, otherwise the variants of `RuleCondition` reference private types.